src/components/MermaidDash.js

```diff
@@ -107,14 +106,14 @@ class MermaidDash extends Component {
 
       const countryCount = this.getCount(updatedSites, 'country_id');
       const projectCount = this.getCount(updatedSites, 'project_id');
-      const userCount = this.getUserCount(updatedSites, projectChoices);
+      const yearCount = this.getUniqueYearCount(updatedSites);
       const uniqueSiteCount = this.getUniqueSiteCount(updatedSites);
       const transectCount = this.getTransectCount(updatedSites);
       const avgCoralCoverCount = this.getAvgCoralCount(updatedSiteProtocols);
 
       if (prevMetricCountriesCount !== countryCount) metrics[0].count = countryCount;
       if (prevMetricProjectsCount !== projectCount) metrics[1].count = projectCount;
-      if (prevMetricUsersCount !== userCount) metrics[2].count = userCount;
+      if (prevMetricUsersCount !== yearCount) metrics[2].count = yearCount;
```

Comment:
Note sure where `prevMetricUsersCount` gets set but it should be changed to reflect `Year`. May want to do a global search for `user` to make sure there aren't other stray bits.